docs/application/toc_all.md

```diff
@@ -1122,6 +1122,7 @@
 #### [Device/Resource Model Manager](/application/tizen-studio/rt-ide/smartths-ext/model-manager.md)
 
 ## Release Notes
+### [3.2 (Feb 20, 2019)](/application/tizen-studio/release-notes/3-2-release-notes.md)
```

Comment:
If I see the Release Note, Tizen Studio 3.2 will be released on March 08, 2019.

Please change the date.